google/ggrc-core
# QUICK-FIX Update lodash and karma

# Issue description

Not the latest versions of `lodash` and `karma` contain vulnerabilities

# Steps to test the changes

From audit perspective:
Run `npm audit`

Actual result: found 210 vulnerabilities
Expected result: found 2 vulnerabilities

From functionality perspective:

Check the diff between `lodash` 4.17.10 and 4.17.11 - https://github.com/lodash/lodash/wiki/Changelog

# Solution description

To avoid 208 vulnerabilities`lodash` and `karma` should be updated. `jQuery` will be updated in scope of https://github.com/google/ggrc-core/pull/9198 and `filemanager-webpack-plugin` has the latest version.

# Sanity checklist




# PR Review checklist
